tchannel/tornado/hyperbahn.py

```diff
@@ -68,7 +66,7 @@ def _advertise(tchannel, service):
             headers={'as': 'json'},
             attempt_times=1,
         )
-    except (ProtocolError, TimeoutError) as e:
+    except Exception as e:  # Big scope to keep it alive.
```

Comment:
Are you sure we should retry on all exceptions? Seems like we should just keep expanding this list as we find additional exceptions.